### DISCUSSION TITLE
How to best notify existing Perspective Github community of upcoming events?

### DISCUSSION THREAD

**Discussion by mindthegab** [perspective-dev/perspective#1365] [General]
Disclaimer: Apologies if this issue is not *strictly* technically related :)

@Andrew-Stein @neilslinger and the Perspective team: we are very much looking forward to the FINOS meetup this upcoming Wednesday featuring Perspective https://www.finos.org/hosted-events/2021-03-31-os-in-fintech-meetup-perspective. 

While we do a lot of promotion to our FINOS general lists, website, social, etc (/cc @grizzwolf) I was wondering what's the best way to promote this to the Perspective specific community, given I don't think we do have a mailing list or a chat channel (of course happy to create one, but that's not necessarily what I am proposing now).

I guess I am thinking about a place not only to promote upcoming events, but also where to link older events (and for example recordings) in a place that is contextual to the project. I feel like the README is nice and clean as it is, so I wouldn't clutter it, but maybe we could add another menu item at https://perspective.finos.org/ called "Events"? 

Curious to hear your thoughts and if this would be useful at all - I get the feeling that witn over 3k stars we could be reaching the Github crowd more efficiently.

**Comment by texodus** [comment]
Yeah it’s a fair point.   Github introduced a “Discussions” feature which we were talking about enabling to move some of the non-actionable or question stuff off of our “Issues” list, how about we start with that?

We had a “Blog” section on the site at one point that functioned something like a news/events/recent updates hub, but I took it down because its infrequent updates (I felt) made it more a monument to our long periods of no-news, rather than a highlight for our releases.

**Reply by texodus** [reply]
... And here we are.  I've got to say, Github pinned discussions sure are _subtle_ ...

<img width="1529" alt="Screen Shot 2021-03-26 at 9 50 48 PM" src="https://user-images.githubusercontent.com/60666/112706872-89452b80-8e7d-11eb-90bb-d00a5d1f68a2.png">
